server/lorax_server/models/flash_phi3.py

```diff
@@ -58,6 +58,19 @@ def __init__(
             trust_remote_code=trust_remote_code,
         )
 
+        try:
+            # Override the tokenizer's eos_token_id with the one from the generation_config
+            # if it is a list or set. We need to do this by adding a new property as the tokenizer
+            # does not officially support multiple eos_token_ids.
+            generation_config = GenerationConfig.from_pretrained(
+                model_id, revision=revision, trust_remote_code=trust_remote_code
+            )
+
+            if isinstance(generation_config.eos_token_id, (list, set)):
+                tokenizer.eos_token_ids = set(generation_config.eos_token_id)
+        except Exception:
+            pass
```

Comment:
Super interesting that the tokenizer only has 1 EOS token ID but the generation config actually maps to a list of token IDs for eos_token_id. I assume this is to support Llama-3 right?

Reply:
For whatever reason both Llama-3 and Phi-3 do this.

Reply:
Oh very strange that they have multiple stop tokens, but good to know!